.github/workflows/xcodegen.yml

```diff
@@ -0,0 +1,21 @@
+name: Xcodegen
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  lint:
```

Comment:
Probably doesn't matter since the GH action will be removed, but is there a reason why the name of the job is `lint`?

Reply:
whoops. It's from your swiftlint.yml :) I used it as a template haha

Reply:
HAHA I see.